Userland/Libraries/LibChess/Chess.cpp

```diff
@@ -9,6 +9,7 @@
 #include <AK/StringBuilder.h>
 #include <AK/Vector.h>
 #include <LibChess/Chess.h>
+#include <ctype.h>
```

Comment:
the functions in AK/CharacterType.h is preferred  to the C library's ctype, since the ones in AK can tolerate utf-8 text.

Reply:
Done!